crates/turbopack-ecmascript/src/analyzer/well_known.rs

```diff
@@ -337,6 +345,99 @@ pub fn require(args: Vec<JsValue>) -> JsValue {
     }
 }
 
+pub fn require_context_require(val: RequireContextValue, args: Vec<JsValue>) -> JsValue {
+    if args.len() < 1 {
```

Comment:
```suggestion
    if args.is_empty() {
```

or

```suggestion
    if args != 1 {
```

considering your comment below.

Reply:
The reason it's not `!= 1` is that a common use-case seems to be
```js
function importAll(r) {
  return r.keys().map(r);
}

const items = importAll(require.context("./"));
```
Where it's passed directly into the `.map` which would add more arguments (the index and array)